modules/ocf_ldap/manifests/init.pp

```diff
@@ -1,33 +1,17 @@
 class ocf_ldap {
+  include ocf::packages::ldapvi
   include ocf_ssl::default_bundle
 
   package { 'slapd':; }
   service { 'slapd':
-    subscribe => File[
-      '/etc/ldap/slapd.conf',
-      '/etc/ldap/schema/ocf.schema',
-      '/etc/ldap/schema/puppet.schema',
-      '/etc/default/slapd',
-      '/etc/ldap/sasl2/slapd.conf'];
+    subscribe => [
+      File['/etc/ldap/krb5.keytab',
+        '/etc/ldap/sasl2/slapd.conf'],
```

Comment:
Yes, mostly because it subscribes to both `File` and `Augeas`, maybe that could be made more clear by making the first file string on a new line?